.gitignore

```diff
@@ -160,6 +160,7 @@ cython_debug/
 #.idea/
 
 # Cache
+cache_dir/*
```

Comment:
redundant. same as `cache_dir/`
@all to remove cached files from `cache_dir` of prev commits to stop pushing `cache_dir/` to remote

Reply:
Updated!